cypress/integration/ambianic-tests/profile-menu.spec.js

```diff
@@ -0,0 +1,41 @@
+/// <reference types="cypress" />
+
+context('Profile menu', () => {
+  before(() => {
+    cy.visit('http://localhost:8080/timeline')
+  })
+
+  it('Should start authentication with Auth0', () => {
+    cy.window().then(win => {
+      win.__store__.dispatch('SAVE_AUTHENTICATED_USER', {
+        user: {
+          email: 'test@mail.com',
+          sub: 'auth0|12121212',
+          name: 'test user'
+        },
+        loadingAuth: false,
+        isAuthenticated: true
+      })
+    })
+
+    cy.get('#vue-tour-button').click()
+    cy.get('[data-cy=profile-toggle]').click()
```

Comment:
@vickywane what assertion are we checking in this test? Code coverage is meaningless if there is no thoughtful behavior validation.